packages/help-panel/src/components/HelpPanel/HelpPanel.tsx

```diff
@@ -7,7 +7,7 @@
 
 import { Color } from '@harness/design-system'
 import React from 'react'
-import { Error, useContentful } from '../../HelpPanelContext'
+import { Error, HelpPanelContext, useContentful } from '../../HelpPanelContext'
```

Comment:
is the import for `useContentful` still needed in this file? If not, we can remove it.

Reply:
removed